### ISSUE TITLE
[BUG] Destroy Private Fields

### ISSUE THREAD

**Issue by BlackYuzia** [javascript-obfuscator/javascript-obfuscator#1056]
## Expected Behavior

You could use private fields  after obfuscation without errors

## Current Behavior

You can't use private fields after obfuscation, but you could did it without (obfuscate).

## Steps to Reproduce

1. Use private fields
2. ~~Lose the keys~~ Get error.

## Your Environment

* Obfuscator version used: 3.4.4
* Node version used: 16.17.1

# Stack trace

## Minimal working example that will help to reproduce issue

```js
class Kruzya {
    #keys; // private, you can't steal my keys!
    #count;

    constructor(name, count = 0) {
        this.#keys = [];
        this.name = name;
        this.#count = count;
    }

    stealKeys() {
        for (const iterator of new Array(this.#count).fill(0)) {
            this.#keys.push(iterator);
        }
    }

    getInfo() {
        console.log("%s, steal [%s] of your keys!", this.name, this.#keys.length);
    }

}

const kruzya = new Kruzya("Kruzya", ~~(Math.random() * 500));
kruzya.stealKeys();
kruzya.getInfo()
```

**Comment by sanex3339** [comment]
Try to evaluate the code above directly in the browser's console. 
I don't see any difference between original and obfuscated code:
```
const g = b;
class Kruzya {
    #keys;
    #count;
    constructor(c, d = 0x0) {
        const e = b;
        this.#keys = [];
        this[e(0x18d)] = c;
        this.#count = d;
    }
    ['stealKeys']() {
        const f = b;
        for (const c of new Array(this.#count)[f(0x187)](0x0)) {
            this.#keys[f(0x18a)](c);
        }
    }
    [g(0x18b)]() {
        const h = g;
        console[h(0x18c)](h(0x189), this['name'], this.#keys['length']);
    }
}
function b(c, d) {
    const e = a();
    b = function (f, g) {
        f = f - 0x187;
        let h = e[f];
        return h;
    };
    return b(c, d);
}
const kruzya = new Kruzya(g(0x188), ~~(Math['random']() * 0x1f4));
kruzya['stealKeys']();
function a() {
    const i = [
        'fill',
        'Kruzya',
        '%s,\x20steal\x20[%s]\x20of\x20your\x20keys!',
        'push',
        'getInfo',
        'log',
        'name'
    ];
    a = function () {
        return i;
    };
    return a();
}
kruzya[g(0x18b)]();

```

**Comment by BlackYuzia** [comment]
hmm, I could give you my current config. Maybe some function do not enabled in your code? Idk what exactly function broke the code.

**Comment by sanex3339** [comment]
It's because `deadCodeInjection` adds some statements with private fields to other parts of the code

**Comment by BlackYuzia** [comment]
meme, so that bug?

**Comment by sanex3339** [comment]
https://github.com/javascript-obfuscator/javascript-obfuscator/pull/1057

**Comment by BlackYuzia** [comment]
@sanex3339 what about not ignore, just add before constructor fields?
like:
```js
class {
 #private
 // ... mb any else?
 constructor() {this.#private = ...}
}
```

Because, If you use any private field(s), that could help you deobfuscate code?

**Comment by sanex3339** [comment]
Such statements might be added to any places outside of a class

**Comment by BlackYuzia** [comment]
Required to test?

**Comment by sanex3339** [comment]
Released as 3.2.5

**Comment by BlackYuzia** [comment]
Yep, fixed
![image](https://user-images.githubusercontent.com/25010528/153382818-7bb30bb6-eba4-4829-bc68-615cc6aef3f7.png)

#1053 trouble isn't fixed yet :c
